Cubical/Homotopy/EilenbergSteenrod.agda

```diff
@@ -0,0 +1,46 @@
+{-# OPTIONS --cubical --no-import-sorts --safe #-}
+
+module Cubical.Homotopy.EilenbergSteenrod where
+
+{-
+This module contains the Eilenberg-Steenrod axioms for ordinary reduced cohomology theories with binary additivity.
+The axioms are based on the ones given in Cavallo's MSc thesis (https://www.cs.cmu.edu/~ecavallo/works/thesis15.pdf) and Buchholtz/Favonia (2018)
+-}
+
+
+open import Cubical.Foundations.Prelude
+open import Cubical.Foundations.Pointed
+open import Cubical.Foundations.Function
+open import Cubical.Foundations.Equiv
+open import Cubical.HITs.Wedge
+open import Cubical.HITs.Pushout
+open import Cubical.HITs.Susp
+
+open import Cubical.Data.Empty
+open import Cubical.Relation.Nullary
+
+open import Cubical.Data.Nat
+open import Cubical.Data.Bool
+open import Cubical.Data.Sigma
+open import Cubical.Data.Int
+
+open import Cubical.Algebra.Group
+open import Cubical.Algebra.AbGroup
+open GroupEquiv
+open GroupHom
+
+record isCohomTheory {ℓ ℓ' : Level} (H : (n : Int) → Pointed ℓ → AbGroup {ℓ'}) : Type (ℓ-suc (ℓ-max ℓ ℓ'))
+  where
+  Boolℓ : Pointed ℓ
+  Boolℓ = Lift Bool , lift true
+  field
+    f* : (n : Int) → {A B : Pointed ℓ} (f : A →∙ B) → AbGroupHom (H n B) (H n A)
+    Suspension : Σ[ F ∈ ((n : Int) {A : Pointed ℓ} → AbGroupEquiv (H (sucInt n) (Susp (typ A) , north)) (H n A)) ]
+                   ({A B : Pointed ℓ} (f : A →∙ B) (n : Int)
+               → fun (f* (sucInt n) (suspFun (fst f) , refl)) ∘ invEq (eq (F n {A = B}))
+                ≡ invEq (eq (F n {A = A})) ∘ fun (f* n f))
+    Exactness : {A B : Pointed ℓ}  (f : A →∙ B) (n :  Int)
+              → Ker _ _ (f* n {A = A} {B = B} f)
+               ≡ Im _ _ (f* n {A = B} {B = _ , inr (pt B)} (cfcod (fst f) , refl))
```

Comment:
Looks like the first two arguments of Ker and Im should indeed be implicit.

Aso, do you need to give A and B in f*? It makes things quite hard to parse...

Reply:
Yeah, agda gets confused about the choice of base point in A and B otherwise:-/

Reply:
Does it help if you change the definition of pointed functions to 
```
_→∙_ : ∀ {ℓ ℓ'} → (A : Pointed ℓ) (B : Pointed ℓ') → Type (ℓ-max ℓ ℓ')
(A , ptA) →∙ (B , ptB) = Σ[ f ∈ (A → B) ] f ptA ≡ ptB
```
in `Cubical.Foundations.Pointed.Base`?